CMake/resolve_dependency_modules/simdjson.cmake

```diff
@@ -13,9 +13,9 @@
 # limitations under the License.
 include_guard(GLOBAL)
 
-set(VELOX_SIMDJSON_VERSION 3.1.5)
```

Comment:
Is this an unrelated upgrade? If yes, please, move it into a separate PR. Otherwise, include it in PR description.